app/components/schedule.hbs

```diff
@@ -15,10 +15,10 @@
     @viewName="agendaDay"
     @defaultView="timelineDay"
     @validRange={{this.validRange}}
-    @now={{moment-format (now) "YYYY-MM-DD"}}
+    @now={{moment-format (now) "DD-MM-YYYY"}}
```

Comment:
This is not user facing format. This needs to be passed in the exact same way to the component